client-registry-api/src/test/java/uk/gov/di/authentication/clientregistry/lambda/UpdateClientConfigHandlerTest.java

```diff
@@ -214,16 +214,16 @@ public void shouldReturn400WhenRequestHasInvalidScope() {
 
     private ClientRegistry createClientRegistry() {
         ClientRegistry clientRegistry = new ClientRegistry();
-        clientRegistry.setClientName(CLIENT_NAME);
-        clientRegistry.setClientID(CLIENT_ID);
-        clientRegistry.setPublicKey("public-key");
-        clientRegistry.setScopes(SCOPES);
-        clientRegistry.setSubjectType("Public");
-        clientRegistry.setRedirectUrls(singletonList("http://localhost/redirect"));
-        clientRegistry.setContacts(singletonList("contant-name"));
-        clientRegistry.setPostLogoutRedirectUrls(singletonList("localhost/logout"));
-        clientRegistry.setServiceType(SERVICE_TYPE);
-        clientRegistry.setClientType(ClientType.WEB.getValue());
+        clientRegistry.withClientName(CLIENT_NAME);
```

Comment:
Can we use the builder functionality here as a refactoring rather than repeated instance calls? Like we have in other uses of the code.

Reply:
Done